airbyte-integrations/connectors/source-facebook-marketing/acceptance-test-config.yml

```diff
@@ -15,12 +15,12 @@ tests:
     - config_path: "secrets/config.json"
       configured_catalog_path: "integration_tests/configured_catalog.json"
       validate_output_from_all_streams: yes
+      # FB serializes numeric fields as strings
```

Comment:
we can do this as a follow up task but we should re-enable schema validation

Reply:
it is already created